lib/math.py

```diff
@@ -0,0 +1,204 @@
+#NOTE: This is ugly.
+from __go__.math import (Pi, E, Ceil, Copysign, Abs, Floor, Mod, Frexp, IsInf,
+    IsNaN, Exp2, Modf, Trunc, Exp, Expm1, Log, Log1p, Log10, Pow, Sqrt, Acos,
+    Asin, Atan, Atan2, Hypot, Sin, Cos, Tan, Acosh, Asinh, Atanh, Sinh, Cosh,
+    Tanh, Erf, Erfc, Gamma, Lgamma) # pylint: disable=g-multiple-import
+
+# Constants
+
+pi = Pi
+
+
+e = E
+
+
+# Number-theoretic and representation functions
+
+def ceil(x):
+    return Ceil(float(x))
+    
+    
+def copysign(x,y):
+    return Copysign(float(x),float(y))
+
+    
+def fabs(x):
+    return Abs(float(x))
+
+
+def factorial(x):
+    
+    def factorial_helper(x, acc):
+        if x <= 1:
+            return acc
+        return factorial_helper(x - 1, acc * x)
+    
+    if x % 1 != 0 or x < 1:
+        raise ValueError
+
+    return factorial_helper(int(x),1)
+
+        
+def floor(x):
+    return Floor(float(x))
+
+
+def fmod(x):
+    return Mod(float(x))
+
+
+def frexp(x):
+    return Frexp(float(x))
+
+
+# NOTE: This function exists in python, but I don't know how to write it,
+#and I don't see it anywhere in Go's math library.
+#
+# def fsum(x):
+#     pass
+
+
+def isinf(x):
+    return IsInf(float(x),0)
+    
+    
+def isnan(x):
+    return IsNaN(float(x))
+
+
+def ldexp(x,i):
+    # NOTE: These are converted to floats, but maybe they should be integers instead?
+    return float(x) * Exp2(float(i))
+    
+    
+def modf(x):
+    #Modf returns (int, frac), but python should return (frac, int)
```

Comment:
Space after # in comments. Also please make comments full sentences ending with period.